vhdbuilder/packer/test/run-test.sh

```diff
@@ -29,24 +29,38 @@ trap cleanup EXIT
 DISK_NAME="${TEST_RESOURCE_PREFIX}-disk"
 VM_NAME="${TEST_RESOURCE_PREFIX}-vm"
 
-if [ "$MODE" == "sigMode" ]; then
-  echo "SIG existence checking for $MODE"
-  id=$(az sig show --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME}) || id=""
-  if [ -z "$id" ]; then
-    echo "Shared Image gallery ${SIG_GALLERY_NAME} does not exist in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
-    exit 1
-  fi
+if [ "$MODE" == "default" ]; then
+  az disk create --resource-group $RESOURCE_GROUP_NAME \
+    --name $DISK_NAME \
+    --source "${OS_DISK_URI}" \
+    --query id
+  az vm create --name $VM_NAME \
+    --resource-group $RESOURCE_GROUP_NAME \
+    --attach-os-disk $DISK_NAME \
+    --os-type $OS_TYPE \
+    --public-ip-address ""
+else 
+  if [ "$MODE" == "sigMode" ]; then
+    id=$(az sig show --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME}) || id=""
+    if [ -z "$id" ]; then
+      echo "Shared Image gallery ${SIG_GALLERY_NAME} does not exist in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
+      exit 1
+    fi
 
-  id=$(az sig image-definition show \
-    --resource-group ${AZURE_RESOURCE_GROUP_NAME} \
-    --gallery-name ${SIG_GALLERY_NAME} \
-    --gallery-image-definition ${SIG_IMAGE_NAME}) || id=""
-  if [ -z "$id" ]; then
-    echo "Image definition ${SIG_IMAGE_NAME} does not exist in gallery ${SIG_GALLERY_NAME} resource group ${AZURE_RESOURCE_GROUP_NAME}"
-    exit 1
-  fi
+    id=$(az sig image-definition show \
+      --resource-group ${AZURE_RESOURCE_GROUP_NAME} \
+      --gallery-name ${SIG_GALLERY_NAME} \
+      --gallery-image-definition ${SIG_IMAGE_NAME}) || id=""
+    if [ -z "$id" ]; then
+      echo "Image definition ${SIG_IMAGE_NAME} does not exist in gallery ${SIG_GALLERY_NAME} resource group ${AZURE_RESOURCE_GROUP_NAME}"
+      exit 1
+    fi
```

Comment:
remvoe the extra space

Reply:
that is not extra space, there is an indentation increase at that level because of how I rewrote the code. I will check again though.